samples/caf_sensor_manager/src/main.c

```diff
@@ -3,20 +3,44 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
+
+#include <zephyr.h>
 #include <app_event_manager.h>
 
 #define MODULE main
-#include <caf/events/module_state_event.h>
 
+
+
+#include <caf/events/module_state_event.h>
 #include <logging/log.h>
+
 LOG_MODULE_REGISTER(MODULE);
 
+#if CONFIG_EVENT_MANAGER_PROXY
```

Comment:
if we make it common we could add this check in header and if proxy is not enabled do inline empty function that is always called from this main